reloader/reloader.py

```diff
@@ -57,6 +63,18 @@ def reload_package(pkg_name, dummy=True, verbose=True):
         dprint("end", fill='-')
 
 
+def reload_dependency(dependency_name):
+    dependency_base = os.path.join(sublime.packages_path(), dependency_name) + os.sep
+
+    for module in list(sys.modules.values()):
+        if getattr(module, '__file__', '').startswith(dependency_base):
+            del sys.modules[module.__name__]
```

Comment:
In my mind, we should not reload `sublime_lib` in this situation.

suppose `foo` requires `bar`, it is necessary to reload `foo` if `bar` is reloaded. However, we could just reload `foo` without issues.

Reply:
What I mean is that if the original call is `reload_package('sublime_lib')`, then `reload_dependency` has to manually `del` the `sublime_lib` modules, because the subsequent call to `reload_package('JSCustom')` won't do it.

Reply:
Would you add a note explaining why we want to the dependency and it’s submodules

Reply:
Done.